airwallex/src/main/java/com/airwallex/android/view/PaymentMethodsActivity.kt

```diff
@@ -149,25 +157,27 @@ class PaymentMethodsActivity : AirwallexCheckoutBaseActivity() {
     }
 
     private fun fetchPaymentMethods() {
-        setLoadingProgress(loading = true, cancelable = false)
-        viewModel.fetchAvailablePaymentMethodTypes().observe(
-            this@PaymentMethodsActivity,
-            { result ->
-                result.fold(
-                    onSuccess = {
-                        setLoadingProgress(loading = false)
-                        // Complete load available payment method type
-                        this.availablePaymentMethodTypes = it
-                        initView(it)
-                        filterPaymentConsents()
-                    },
-                    onFailure = {
-                        setLoadingProgress(loading = false)
-                        alert(message = it.message ?: it.toString())
-                    }
-                )
-            }
-        )
+        CoroutineScope(Dispatchers.Main).launch {
```

Comment:
minor nitpick: I'm seeing a `Lambda argument should be moved out of parentheses ` warning here. Just need to take the closure out of the brackets

Reply:
There's another one in `fetchPaymentMethods` too

Reply:
Cool, I guess you meant another in `deletePaymentConsent`, I'll refactor both